index.js

```diff
@@ -57,10 +60,27 @@ module.exports = function (options) {
 		}
 
 		try {
-			var errors = checker.checkString(file.contents.toString(), file.relative);
+			var fixResults;
+			var errors;
+			var changed = false;
```

Comment:
@sindresorhus We are in the process of removing this, and will update the PR. What gulp plugin would you recommend to sit between `jscs` and `gulp.dest` so not all files that are processed by jscs are rewritten every time it runs? Or do you suggest using `gulp-changed` to filter the input stream will be sufficient and further optimization of the output isn't needed?

The original intent of this code was to allow filtering of the files that were modified by jscs during this process if desired.